Electronic Colloquium on Computational Complexity.js

```diff
@@ -207,17 +189,14 @@ var testCases = [
 				"date": "2023/5/30",
 				"abstractNote": "One of the major open problems in complexity theory is proving super-logarithmic lower bounds on the depth of circuits (i.e., $\\mathbf{P}\\not\\subseteq \\mathbf{NC}^{1}$). Karchmer, Raz, and Wigderson (Computational Complexity 5(3/4), 1995) suggested to approach this problem by proving that depth complexity of a composition of functions $f \\diamond g$ is roughly the sum of the depth complexities of $f$ and $g$. They showed that the validity of this conjecture would imply that $\\mathbf{P}\\not\\subseteq\\mathbf{NC}^{1}$.The intuition that underlies the KRW conjecture is that the composition $f \\diamond g$ should behave like a \"direct-sum problem\", in a certain sense, and therefore the depth complexity of $f \\diamond g$ should be the sum of the individual depth complexities. Nevertheless, there are two obstacles toward turning this intuition into a proof: first, we do not know how to prove that $f \\diamond g$ must behave like a direct-sum problem; second, we do not know how to prove that the complexity of the latter direct-sum problem is indeed the sum of the individual complexities.In this work, we focus on the second obstacle. To this end, we study a notion called ``strong composition'', which is the same as $f \\diamond g$ except that it is forced to behave like a direct-sum problem. We prove a variant of the KRW conjecture for strong composition, thus overcoming the above second obstacle. This result demonstrates that the first obstacle above is the crucial barrier toward resolving the KRW conjecture. Along the way, we develop some general techniques that might be of independent interest.",
 				"archiveID": "TR23-078",
-				"libraryCatalog": "Electronic Colloquium on Computational Complexity",
+				"language": "en",
+				"libraryCatalog": "eccc.weizmann.ac.il",
 				"repository": "Electronic Colloquium on Computational Complexity",
 				"shortTitle": "Toward Better Depth Lower Bounds",
 				"url": "https://eccc.weizmann.ac.il/report/2023/078/",
 				"attachments": [
 					{
-						"title": "ECCC Snapshot",
```

Comment:
Using the Embedded Metadata translator means that we don't get a snapshot anymore. Should it be added back?

Reply:
I don't think the Snapshot adds much here -- people will principally want the PDF and already get the abstract in the metadata, so I'd leave it out.